package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
We would anyway pull all the dependencies when building, so using `prepackaged node modules` or `submoduled with dependencies` in source would have the same effect. Maybe we could trim all the dev dependencies from eslint?